src/libraries/System.Private.DeveloperExperience.Console/src/System.Private.DeveloperExperience.Console.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
We should not need Windows-specific version anymore

Reply:
Seems to be needed:

.packages/microsoft.dotnet.build.tasks.targetframework.sdk/5.0.0-beta.20374.1/build/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk.targets(64,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Not able to find a compatible supported target framework for net5.0 in Project System.Console.csproj. The Supported Configurations are net5.0-Windows_NT, net5.0-Unix, net5.0-Android, net5.0-iOS, net5.0-tvOS, net5.0-Browser

Reply:
Makes sense - now that it is symmetric between Windows and Unix